renderer/utils/utils.ts

```diff
@@ -86,7 +90,23 @@ export function parseEventTitle(event, availableProjects: Array<string>) {
       break;
 
     case 1:
-      event.description = items[0];
+      for (let i = 0; words.length > i; i++) {
+
+        if (allProjects.includes(words[i].toLowerCase())) {
+          event.project = words[i].toLowerCase();
+
+          for (let j = 0; words.length > j; j++) {
+            const activities = latestProjAndAct[words[i].toLowerCase()];
+            if (activities && activities.includes(words[j].toLowerCase())) {
+              event.activity =
+                activities[activities.indexOf(words[j].toLowerCase())];
+            }
+          }
+          break;
```

Comment:
Only as idea, can we declare some vars at the beggining like:
const word = words[i].toLowerCase() to avoid duplicating